dvc/external_repo.py

```diff
@@ -4,11 +4,11 @@
 import tempfile
 from contextlib import contextmanager
 from distutils.dir_util import copy_tree
-
```

Comment:
Please don't remove this, it separates std from 3rd party imports. And, well, touching unrelated lines is no bueno in general 🙂

Reply:
@efiop deepsource asked me to do it :-X

Reply:
@maykulkarni It is purely advisory, no need to make it fully pass.